src/blocks/weather.rs

```diff
@@ -499,6 +521,22 @@ fn need_forecast(format: &Format, format_alt: Option<&Format>) -> bool {
     has_forecast_key(format) || format_alt.is_some_and(has_forecast_key)
 }
 
+fn calculate_sunrise_sunset(lat: f64, lon: f64) -> Result<(DateTime<Utc>, DateTime<Utc>)> {
+    let tz: Tz = tz_search::lookup(lat, lon)
+        .or_error(|| format!("Unable to find tz for lat/long {}, {}", lat, lon))?
+        .parse()
+        .error("Unable to parse tz")?;
+    let date = Utc::now().with_timezone(&tz);
+    let (sunrise, sunset) = sunrise_sunset(lat, lon, date.year(), date.month(), date.day());
```

Comment:
Is it really necessary to perform a timezone lookup? Firstly, are you sure that year/month/day are supposed to be in local time (it is not documented unfortunately)? If it is, is the difference between adjacent days that noticeable?

Reply:
Actually, I suspect your right since the value is being passed into `Utc.with_ymd_and_hms` here.
Let me change that to use the current Utc date.